### PR TITLE
Verification

### PR DESCRIPTION
**Pull request by Jakob37**
Not sure how to document this

This branch contains:

* Intersect bed file updates
* Sentieon version update

It will also contain:

* VEP update

# Description

What is changed? What is the result of the changes?

fixes # (issue)

## Type of change

- [x] Major change 
- [ ] Minor change
- [ ] Patch
- [ ] Documentation

# Checklist:

- [x] My code follows the [style guidelines for NF pipelines at CMD](http://mtlucmds1.lund.skane.se/wiki/doku.php?id=nextflow&s[]=nextflow#code_style_at_cmd)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings (Keep an eye on `.nextflow.log` !)
- [x] I have updated the CHANGELOG
- [ ] The latest commit in the master branch is tagged
      with the correct version number

Points from startup discussion

- [x] Remove Gnomad from PR (@Jakob37 )
- [x] Merge updated rank model (@alkc )
- [x] First sanity testing (@alkc )
- [x] Update checklist templates for verification (@alkc )
- [x] Update verification document (@Jakob37 )
- [x] Document groups of causative variants linked to each risk profile (find a good CMD-only spot and put document there)
- [x] Carry out risk analysis (and include in checklist template) (@alkc )
- [ ] Gather test scripts in a separate repo
- [x] GIAB single test
- [x] GIAB trio test
- [x] Dry run - Blueprint variants
- [x] Known causative variants (@Jakob37 )

## Major change
- [x] Stub run completes without errors or new warnings
- [x] `onco` run finishes without any new warnings/errors and the results can 
       be loaded into scout
- [x] `wgs` single run finishes without any new warnings/errors and the results 
       can be loaded into scout
- [x] `wgs` trio run finishes without any new warnings/errors and the results 
       can be loaded into scout
- [x] At least one other person has reviewed and approved my code
- [x] I have made corresponding changes to the documentation

## Minor change
- [ ] Stub run completes without errors or new warnings
- [ ] `onco` run finishes without any new warnings/errors and the results can 
       be loaded into scout
- [ ] `wgs` single run finishes without any new warnings/errors and the results 
      can be loaded into scout
- [ ] `wgs` trio run finishes without any new warnings/errors and the results 
       can be loaded into scout
- [ ] At least one other person has reviewed and approved my code
- [ ] I have made corresponding changes to the documentation

## Patch
- [ ] Stub run completes without errors or new warnings

## [Optional] Are there any post-deployment tasks we need to perform?

- [x] Deploy new rank models
 
# Instructions for the reviewers

## How to test the changes

1. Step 1
2. Step 2

## [Optional] Additional information

# Review

## Review performed by:
- Reviewer 1: Name1, (\@Github\_handle1)  
- Reviewer 2: Name1,  (\@Github\_handle2)
- ...
    
## Testing performed by:
- Name, (\@Github\_handle)
- ...

# Post-merge
- [ ] The `master` branch has been tagged with the new version number.

### CODE REVIEW

**Comment by Jakob37** [comment]
I cleaned up the tags and the CHANGELOG

File: configs/nextflow.hopper.config [hunk 1]
```diff
@@ -79,6 +80,9 @@ params {
 
   // SLURM MISC //
   queue             = 'normal'
+
+  // CONTAINERS //
+  container_fastp = '/fs1/resources/containers/fastp_0.23.4.sif'
```
Comment (alkc) [inline configs/nextflow.hopper.config:85]: `sentieon_container` vs `container_fastp`. 

Nitpick but we should remember to harmonize those vars when prepping for release.

Comment (Jakob37) [reply configs/nextflow.hopper.config:85]: Yes

File: configs/nextflow.hopper.config [hunk 2]
```diff
@@ -114,11 +118,9 @@ profiles {
     params.distributed = false
     params.intervals = "PH"
     params.bedgz = "PH"
-    params.panelhs = "PH"
-    params.panelhs2 = "PH"
     // RANK MODELS // 
-    params.rank_model_s = "/fs1/resources/scout/rank_models_38/rank_model_v5.2.1_single.ini"
-    params.rank_model = "/fs1/resources/scout/rank_models_38/rank_model_v5.2.1.ini"
+    params.rank_model_s = "/fs1/jakob/resources/grpmax_rank_models/rank_model_v5.2.1_single.ini"
```
Comment (Jakob37) [inline configs/nextflow.hopper.config:122]: This must be fixed before finishing up. This is the same rank model, but targeting the CSQ grpmax field instead. I guess we will bunch together all rank model changes somehow (as I think we discussed in the RD meeting)

Comment (alkc) [reply configs/nextflow.hopper.config:122]: Yes, when everything else is finished

File: newdocs/list_of_all_used_software.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+| Tool                   | Version      | Type        | URL                                                    | External contact                                       | Internal contact                      | Short description                                                                         |
```
Comment (Jakob37) [inline newdocs/list_of_all_used_software.md:1]: This file is outdated, we need to transfer the version updates to the `docs` one

File: configs/nextflow.hopper.config (hunk 2 above)
Comment (Jakob37) [reply configs/nextflow.hopper.config]: Removed now, as genmod no longer is used here

**Comment by Jakob37** [comment]
Gnomad and grpmax references are removed now from this PR.

It should be ready for further testing (when the rank models are in).

File: configs/nextflow.hopper.config (hunk 1 above)
Comment (Jakob37) [reply configs/nextflow.hopper.config:91]: Renamed to `container_sentieon`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
 # CHANGELOG
 
+### [Unreleased]
+	
+* Give the Sentieon container path by a parameter in the config file
+* Update the Sentieon container to 202308 version
+* Split out the `sentieon_qc` post-processing into its own process `sentieon_qc_postprocess`
+* Update the Perl script used in `sentieon_qc_postprocess` to take input parameters as explicit arguments
+* Update intersect file to latest used version of ClinVar (20231230)
+* Update fastp version (0.23.4)
```
Comment (Jakob37) [inline CHANGELOG.md:10]: Duplicate of the line below

**Pull request merged**